### PR TITLE
Bs tl/add filters#180875319

### PR DESCRIPTION
**Pull request by tommylusun-au**
# Changes

Add label filter dropdowns to LabelFilter component if exists.

image

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [x] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/main/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: packages/components/src/components/LabelFilterDropdowns/LabelFilterDropdowns.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          return {
+            id: item,
+            text: item
+          };
+        })}
+        itemToString={item => {
+          return item.text;
+        }}
+        onChange={({ selectedItem }) => {
+          handleDropdownChange(`${label}:${selectedItem.id}`);
```
Comment (Nick-Geofroy) [inline packages/components/src/components/LabelFilterDropdowns/LabelFilterDropdowns.js:79]: can we just pass in label and selectedItem.id as 2 params to `handleDropdownChange`? that way we don't need to do the split on :

**Review by Nick-Geofroy — APPROVED** [review]
minor comments but otherwise LGTM

File: packages/components/src/components/LabelFilterDropdowns/LabelFilterDropdowns.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          }
+        }
+      });
+    }
+  });
+  const labelsList = Object.keys(labelFilters);
+
+  const selectedDropdownValues = {};
+  labelsList.forEach(label => {
+    const selected = filters.find(filter => {
```
[the remark this answers is not in the document]
Comment (tommylusun-au) [reply packages/components/src/components/LabelFilterDropdowns/LabelFilterDropdowns.js:54]: hmmm this is autoformated based on their prettier settings (it throws errors on the UI if any formatting is wrong). which part is spaced different?

File: packages/components/src/components/LabelFilterDropdowns/LabelFilterDropdowns.js (hunk 1 above)
Comment (tommylusun-au) [reply packages/components/src/components/LabelFilterDropdowns/LabelFilterDropdowns.js]: updated, the colon was for an earlier implementation to match the search bar (requires the `:` format).
the handleFilters functions still require the param to be in the format `label=value`, but moved it into the function definition now

**Pull request merged**